tensorflow_datasets/image/scene_parse_150.py

```diff
@@ -0,0 +1,102 @@
+"""MIT Scene Parsing Benchmark (SceneParse150)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+# BibTeX citation
+_CITATION = """
+@inproceedings{zhou2017scene,
+    title={Scene Parsing through ADE20K Dataset},
+    author={Zhou, Bolei and Zhao, Hang and Puig, Xavier and Fidler, Sanja and Barriuso, Adela and Torralba, Antonio},
+    booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
+    year={2017}
+}
+@article{zhou2016semantic,
```

Comment:
Why are you added 2016 citation. You use 2017 data.

Reply:
Removed the old one.